nifi-maven-archetypes/nifi-processor-bundle-archetype/src/main/resources/archetype-resources/nifi-__artifactBaseName__-processors/src/main/java/MyProcessor.java

```diff
@@ -41,7 +51,8 @@
 public class MyProcessor extends AbstractProcessor {
 
     public static final PropertyDescriptor MY_PROPERTY = new PropertyDescriptor
-            .Builder().name("My Property")
+            .Builder().name("MY_PROPERTY")
```

Comment:
Not that it matters, but most of the time I see machine-friendly names I've been seeing them like "my-processor-my-property" vs "MY_PROPERTY". Since it's a placeholder here, then no big deal, just sharing :)